### PR TITLE
add script

### PR DESCRIPTION
**Pull request by felipesu19**
fixes: https://github.com/github/vscode-github-actions/issues/131

Depends on: https://github.com/actions/languageservices/pull/32 being merged in and then reconciling the changes in publish.yml 

Based on: https://github.com/github/vscode-github-actions/pull/150

harder to test however because there have been no changes merged in recently, this is what the script outputs now: 
Screenshot 2023-04-27 at 1 06 43 PM

### CODE REVIEW

**Review by lrotschy — DISMISSED** [review]

**Comment by felipesu19** [comment]
Shown to work: https://github.com/actions/languageservices/pull/38
https://github.com/actions/languageservices/actions/runs/4863895164/jobs/8672215423

File: script/workflows/generate-release-notes.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# store the release notes in a variable so we can use it later
+
+echo "Release $NEW_RELEASE" >> releasenotes.md
+
+echo $CHANGELIST | jq -r '.[].title' | while read line; do
+  echo " - $line" >> releasenotes.md
+done
+
+echo " "
```
Comment (jtamsut) [inline script/workflows/generate-release-notes.sh:30]: nit: Add a newline here

File: script/workflows/generate-release-notes.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# this script is used to generate release notes for a given release
+# first argument is the pull request id for the last release
+# the second is the new release number
+
+# the script then grabs every pull request merged since that pull request
+# and outputs a string of release notes
+
+echo "Generating release notes for $2"
```
Comment (jtamsut) [inline script/workflows/generate-release-notes.sh:9]: another nit (feel free to ignore): You are defining a variable below `LAST_RELEASE` but not using it above. `$2` is sort of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).

Comment (felipesu19) [reply script/workflows/generate-release-notes.sh]: Fixed!

**Pull request merged**